src/mono/wasm/runtime/diagnostics/index.ts

```diff
@@ -94,12 +94,25 @@ export const diagnostics: Diagnostics = getDiagnostics();
 let suspendOnStartup = false;
 let diagnosticsServerEnabled = false;
 
-export async function mono_wasm_init_diagnostics(options: DiagnosticOptions): Promise<void> {
+let diagnosticsInitialized = false;
+
+export async function mono_wasm_init_diagnostics(opts: "env" | DiagnosticOptions): Promise<void> {
```

Comment:
is the signature with `DiagnosticOptions` param type still useful ?

Reply:
No. I want to do a follow up PR to simplify more of the diagnostics support. removing this argument is just one piece.